proto/data.go

```diff
@@ -546,3 +546,91 @@ type Int32Slice []int32
 func (s Int32Slice) Len() int           { return len(s) }
 func (s Int32Slice) Swap(i, j int)      { s[i], s[j] = s[j], s[i] }
 func (s Int32Slice) Less(i, j int) bool { return s[i] < s[j] }
+
+// ToInternal places the datapoints in a TimeSeriesData message into one or
+// more InternalTimeSeriesData messages. The structure and number of messages
+// returned depends on two variables: a key duration, and a sample duration.
+//
+// The sample duration is an interval length in nanoseconds; datapoints which
```

Comment:
Reorder explanations for sample and key durations because that's the order they're introduced in the preceding paragraph.